app/webpack/base.config.js

```diff
@@ -46,6 +46,7 @@ module.exports = {
       routes: path.resolve(__dirname, '../src/routes'),
       layouts: path.resolve(__dirname, '../src/layouts'),
       'react-intl': path.resolve(__dirname, '../node_modules/react-intl/dist/react-intl.js'), // https://github.com/formatjs/react-intl/issues/1499#issuecomment-570151879
+      ga4Events: path.resolve(__dirname, '../src/components/main/analytics/events/ga4Events'),
```

Comment:
Better called it like `events` that's confusing at the first glance `events: path.resolve(__dirname, '../src/components/main/analytics/events/ga4Events'),` but it provide some advantages like in whole app we have reference to `events` folder after removing ga3 we are just change path here in one place and second is we abstract from ga naming or version because it is just an events in general
@AmsterGet pls check this comment